ts/sagas/messages.ts

```diff
@@ -0,0 +1,239 @@
+/**
+ * Generators to manage messages and related services.
+ */
+
+import { none, Option, some } from "fp-ts/lib/Option";
+import {
+  BasicResponseType,
+  TypeofApiCall
+} from "italia-ts-commons/lib/requests";
+import { Task } from "redux-saga";
+import {
+  all,
+  call,
+  cancel,
+  cancelled,
+  Effect,
+  fork,
+  put,
+  select,
+  take
+} from "redux-saga/effects";
+
+import { Messages } from "../../definitions/backend/Messages";
+import { MessageWithContent } from "../../definitions/backend/MessageWithContent";
+import { ServicePublic } from "../../definitions/backend/ServicePublic";
+import {
+  BackendClient,
+  BackendClientT,
+  GetMessageT,
+  GetServiceT
+} from "../api/backend";
+import { apiUrlPrefix } from "../config";
+import {
+  MESSAGES_LOAD_CANCEL,
+  MESSAGES_LOAD_REQUEST
+} from "../store/actions/constants";
+import {
+  loadMessagesCancel,
+  loadMessagesFailure,
+  loadMessagesSuccess,
+  loadMessageSuccess,
+  MessagesLoadCancel,
+  MessagesLoadRequest
+} from "../store/actions/messages";
+import { loadServiceSuccess } from "../store/actions/services";
+import { messagesByIdSelectors } from "../store/reducers/entities/messages/messagesById";
```

Comment:
is this plural because it implements many selectors?

Reply:
No, just a typo